### PR TITLE
ADD about.html

### PR DESCRIPTION
**Pull request by wuyuhang422**
add some REAL FACTS.
Pictures in about.html may not work,you can reset paths to images/

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+.ty-preferences .nav-group-item.active {
+    color: white;
+    background: #999;
+}
+
+ .typora-export li, .typora-export p, .typora-export,  .footnote-line {white-space: normal;} 
+</style>
+</head>
+<body class='typora-export' >
+<div  id='write'  class = 'is-mac'><h2><a name="欢迎大家和我交流" class="md-header-anchor"></a><span>欢迎大家和我交流</span></h2><p><span>我，Qingyu，SDOIer，初三，山东OI&amp;MO现役省队。</span></p><p><span>我对 SRM hard 的评价：</span></p><p><img src="images/1.jpg" referrerpolicy="no-referrer" alt="1"></p><p><span>我对 ZJOI Round 的评价：</span></p><p><img src="images/2.jpg" referrerpolicy="no-referrer" alt="2"></p><p><span>欢迎大家和我交流学习 一同进步！我会将我的新发现不定时上传到这里！</span></p></div>
```
Comment (wuyuhang1) [inline about.html:507]: The picture may can't be loadad

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# CSQingyu.github.io
+这里是高考数学 145 英语 150 SDOI&MO 队爷 Qingyu 的博客
```
Comment (wuyuhang1) [inline README.md:2]: I think you should use English to express this FACTS.

File: about.html (hunk 1 above)
Comment (slarysct) [reply about.html:507]: > The picture may can't be loadad

Why？I'm a new to Github. I think this picture will be loaded successfully.
But the sentence here is really confusing to me.
I must use google translate to understand it.
> My comments on SRM hard:
I think SRM hard is really HARD for me ,so  can u get in touch with me?
I admire you so much.

Comment (slarysct) [reply about.html:507]: But I think this pr is corrected

File: README.md (hunk 2 above)
Comment (slarysct) [reply README.md:2]: Amazing! Thanks to @wuyuhang422  let me to know how GREAT Qinyu is.
I think The university entrance exam in China is really really hard for me.In my country, I often hear people say that the college entrance examination in China is difficult.
I think he should go to THU,or MIT.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+欢迎大家来膜拜！
```
Comment (pretesro) [inline README.md:3]: README is too short.
It's not enouth to reflect how @CSQingyu is strong.

File: README.md (hunk 2 above)
Comment (pretesro) [inline README.md:2]:
```suggestion
This is Qingyu's blog, whose English for college entrance examination got 150, math got 145. The provincial team in the OI/MO .
这里是高考数学 145 英语 150 SDOI&MO 队爷 Qingyu 的博客
He has won many gold MEDALS in the provincial games and mastered a lot of knowledge from university.
他在省级比赛中获得了很多金牌，并掌握了很多大学的知识
```

File: about.html (hunk 1 above)
Comment (pretesro) [inline about.html:507]:
```suggestion
<div  id='write'  class = 'is-mac'><h2><a name="欢迎大家和我交流" class="md-header-anchor"></a><span>欢迎大家和我交流</span></h2><p><span>我，Qingyu，SDOIer，初三，山东OI&amp;MO现役省队。</span></p><p><span>我对 SRM hard 的评价：</span></p><p><img src="images/1.jpg" referrerpolicy="no-referrer" alt="1"></p><p><span>我对 ZJOI Round 的评价：</span></p><p><img src="images/2.jpg" referrerpolicy="no-referrer" alt="2"></p><p><span>欢迎大家和我交流学习 一同进步！I will study hard and win the gold medal as soon as possible.</span></p></div>
```

File: about.html [hunk 4]
```diff
@@ -0,0 +1,509 @@
+<!doctype html>
```
Comment (pretesro) [inline about.html:1]:
```suggestion
<!DOCTYPE HTML>
```

File: README.md (hunk 2 above)
Comment (pretesro) [reply README.md:2]: > I think you should use English to express this FACTS.

That's what I did. You know , this writer thinks that his English is poor.
I have translated it by using GOOGLE, and I have fixed some small details.
Thanks you for your suggestions.

File: about.html (hunk 1 above)
Comment (pretesro) [reply about.html:507]: > The picture may can't be loadad

Oh, I wiil upload this again.


feel free to ask me other things about how Qingyu is strong ,if u have some suggesstions you can contact me.

Comment (wuyuhang422) [reply about.html:507]: > > The picture may can't be loadad
> 
> Oh, I wiil upload this again.
> 
> 
> feel free to ask me other things about how Qingyu is strong ,if u have some suggesstions you can contact me.

Thank you for your contributions.

**Comment by wuyuhang422** [comment]
> SUCH A AMAZING PR!A ton of AMAZING features ! AND a log of NEW FACTS!
> Thank you for your contribution.

Thank you ,I will do it better.

**Comment by wuyuhang422** [comment]
> Thanks you, @wuyuhang422 !
> I don't think the author thought twice about introducing himself when he first set up his website,or maybe he's too modest.
> now I'm realized that This author is the person I am looking for ---- Someone I can worship.
> Expect better pull requests.

Yeah,I think too ---

**Pull request closed without merging**